### PR TITLE
Update pupa clean so that Jurisdiction, Division, and Post objects are not deleted

### PR DESCRIPTION
**Pull request by hancush**
## Description

See title. Divisions and Posts are unchanging and should not be removed by the `clean` command!

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
       entry_points='''[console_scripts]
 pupa = pupa.cli.__main__:main''',
       install_requires=[
-          'Django>=2.2',
+          'Django>=2.2,<5',
```
Comment (hancush) [inline setup.py:21]: `index_together` was removed in Django 5.

File: pupa/tests/django_settings.py [hunk 2]
```diff
@@ -7,11 +9,11 @@
...
-        'NAME': 'test',
-        'USER': 'test',
-        'PASSWORD': 'test',
-        'HOST': 'localhost',
-        'PORT': 5432,
+        'NAME': os.getenv('POSTGRES_DB', 'test'),
+        'USER': os.getenv('POSTGRES_USER', 'test'),
+        'PASSWORD': os.getenv('POSTGRES_PASSWORD', 'test'),
+        'HOST': os.getenv('POSTGRES_HOST', 'localhost'),
+        'PORT': os.getenv('POSTGRES_PORT', 5432),
```
Comment (hancush) [inline pupa/tests/django_settings.py:16]: Allow for setting these as environment variables, useful for, e.g., if you are running Postgres in a container.

File: pupa/tests/clean/test_clean.py [hunk 3]
```diff
@@ -118,23 +134,25 @@ def test_clean_command_failsafe(subparsers, organization, person):
     for p in stale_people:
         p.memberships.create(organization=organization)
 
+    cmd = CleanCommand(subparsers)
```
Comment (hancush) [inline pupa/tests/clean/test_clean.py:137]: pytest complained when this was instantiated multiple times.

**Review by antidipyramid — APPROVED** [review]
Very cool!

**Pull request merged**